static/io/include/mapbox/io.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <ghc/filesystem.hpp>
+#include <nonstd/expected.hpp>
+
+#include <string>
+
+namespace mapbox {
+namespace base {
+
+using ErrorType = std::string;
```

Comment:
`mapbox::base::ErrorType` looks quite generic, are we going to use it outside of io module? if no, let's maybe return just `nonstd::expected<std::string, std::string>` of define a `io` namespace ``mapbox::base::io::ErrorType``

Reply:
:+1: for adding a namespace of its own for io e.g. `mapbox::base::io`